atlassian/jira.py

```diff
@@ -1867,6 +1864,20 @@ def set_issue_status(self, issue_key, status_name, fields=None, update=None):
             data["update"] = update
         return self.post(url, data=data)
 
+    def get_issue_status_changelog(self, issue_id):
+        # Get the issue details with changelog
+        issue_id = self.get_issue(issue_id, expand="changelog")
```

Comment:
Do not override `issue_id`, name it `response`.

Reply:
@gkowalc  could you commit into that PR please

Reply:
done, I implemented both suggestions.